config/sets/sylius/plus/multi-source-inventory-plugin.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+declare(strict_types=1);
+
+use Rector\Config\RectorConfig;
+use Sylius\SyliusRector\Rector\Class_\AddInterfaceToClassExtendingTypeRector;
+use Sylius\SyliusRector\Rector\Class_\AddTraitToClassExtendingTypeRector;
+
+return static function (RectorConfig $rectorConfig): void {
+    $rectorConfig->ruleWithConfiguration(AddInterfaceToClassExtendingTypeRector::class, [
+        'Sylius\Component\Core\Model\ProductVariant' => [
+            'Sylius\MultiSourceInventoryPlugin\Domain\Model\ProductVariantInterface',
+        ],
+        'Sylius\Component\Core\Model\Shipment' => [
+            'Sylius\MultiSourceInventoryPlugin\Domain\Model\ShipmentInterface',
+        ],
+    ]);
+
+    $rectorConfig->ruleWithConfiguration(AddTraitToClassExtendingTypeRector::class, [
+        'Sylius\Component\Core\Model\ProductVariant' => [
+            'Sylius\MultiSourceInventoryPlugin\Domain\Model\InventorySourceStocksAwareTrait',
```

Comment:
Is it enough? Because in the end application we've also initialized the trait in constructor:
```
    use InventorySourceStocksAwareTrait {
        __construct as private initializeProductVariantTrait;
    }

    public function __construct()
    {
        parent::__construct();

        $this->initializeProductVariantTrait();
    }
```